libs/net/dal/Migrations/1.0.164/Down/PostDown/00-NotificationTemplate.sql

```diff
@@ -0,0 +1,56 @@
+DO $$
```

Comment:
Down is for rollback.  Sometimes we don't need any special steps to rollback because the change is not destructive.  Is this rollback required?  It doesn't appear to do anything.